todo_app/templates/index.html

```diff
@@ -56,22 +67,26 @@ <h2>Items</h2>
         <summary>Older Completed Items</summary>
         {%for item in view_model.older_done_items %}        
         <li class="list-group-item">       
-           {{item.title}} - {{item.status}} <a href="{{url_for('delete_item', id=item.id)}}"><i class="fas fa-trash-alt"></i></a><a href="{{url_for('uncomplete_item', id=item.id)}}" class="btn btn-primary btn-sm m-1" role="button" aria-pressed="true">To Do</a>
+           {{item.title}} - {{item.status}} 
+           {% if isWriter==True: %}
+           <a href="{{url_for('delete_item', id=item.id)}}"><i class="fas fa-trash-alt"></i></a><a href="{{url_for('uncomplete_item', id=item.id)}}" class="btn btn-primary btn-sm m-1" role="button" aria-pressed="true">To Do</a>
```

Comment:
Splitting these separate links over multiple lines would make this a lot easier to read